### DISCUSSION TITLE
how to config Erasure code setting ?

### DISCUSSION THREAD

**Discussion by rickyzhuang** [minio/minio#20269] [Q&A]
in case that ,I have 3 nodes ,each node has 2 drivers . 
how to set the Erasure code setting,for example  to EC:4:2  ?
and does this config be available  for a bucket or for the  all clusters

**Comment by klauspost** [comment]
The default will set up a set of 6 drives. Here is how you set the `EC:2` parity: https://min.io/docs/minio/linux/reference/minio-server/settings/storage-class.html#standard-storage-class
